bundles/pom.xml

```diff
@@ -1,690 +1,2706 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Something weird happend to this file, please revert / only change the line to add this binding.